lib/zone-spec/long-stack-trace.ts

```diff
@@ -6,6 +6,8 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {zoneSymbol} from '../common/utils';
```

Comment:
Can't do imports like that, because it would pull in the whole utils file. This file is meant to be stand alone and so no external imports are allowed.

Just use `Zone.__symbol__`